### PR TITLE
Initial Supybot install

### PR DESCRIPTION
**Pull request by djmitche**
This is currently running in the supybot.buildbot.net jail.  Remaining to do:

* only restart the supervisord service when something changes (or maybe use a supybot command to reload)
* add a user (not sure this is necessary?)
* adjust the configuration to suit our needs
* install meetbot

### CODE REVIEW

**Comment by sa2ajj** [comment]
I think a better approach would be to have tarball of supybot we'd like to use.

**Comment by djmitche** [comment]
Where would we put the tarball?  I cloned the repo since you had forked it.  Could just as easily use the upstream github repo.

File: jail-supybot.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    master_service_name: "supybot"
+
+  roles:
+  - base
+  - role: user
+    user_id: "{{ worker_account }}"
+    user_name: Buildbot Worker Account
+  - role: virtualenv
+    venv_user: "{{ worker_account }}"
+    venv_home_dir: "{{ getent_passwd[worker_account].4 }}"
```
Comment (seankelly) [inline jail-supybot.yml]: Discovered recently the `expanduser` filter in ansible. It will expand paths with tildes in them. This includes the form "~user". I don't think we have a variable for this (maybe `worker_home` being ~bb-infra), but I think it would be a comprehension improvement over the `getent_paswd`.

Since it appears in multiple places, might be useful to have a separate PR for that.

Comment (djmitche) [inline jail-supybot.yml]: Agreed -- want to note that down as a Trac bug if you don't have a chance to do a PR?

Comment (sa2ajj) [inline jail-supybot.yml]: I _think_ I tried to use `~user` at some point and it just did not work.

If it works, I'd gladly get rid of `getent`.

Comment (sa2ajj) [inline jail-supybot.yml]: [trac #3279](http://trac.buildbot.net/ticket/3279)

**Comment by djmitche** [comment]
OK, updated to install and run MeetBot.  I don't really see the point of using a tarball over a git repository, but I'm happy to do so if you tell me where to park the tarball.

Aside from that, this is working and ready to run a meeting.  Setting up a web service for the resulting meetings can happen in another PR.

File: roles/supybot/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    dest: "{{ supybot_user_home }}/conf/{{ item }}"
+    state: touch
+    owner: "{{ supybot_user }}"
+  with_items:
+    - channels.conf
+    - ignores.conf 
+    - userdata.conf
+    - users.conf
+
+# TODO: only when changed
```
Comment (seankelly) [inline roles/supybot/tasks/main.yml]: The `notify` action that gets triggered can do anything, including running a shell script or command. To make this only happen when something changes, find the steps that indicate something changed, have them notify `restart supybot` and move this step to `handlers/main.yml` with the same name.

Comment (sa2ajj) [inline roles/supybot/tasks/main.yml]: :+1: for the suggestion
